TW-121 updates for compose command (#1749)
temporalio/documentation: docs-src/tctl-v1/how-to-install-tctl.md

You can install [tctl](/tctl-v1) in the following ways.

- Install locally by using [Homebrew](https://brew.sh/): `brew install tctl`

> - Run locally together with Temporal Server in [Docker Compose](https://github.com/temporalio/docker-compose): `docker exec temporal-admin-tools tctl YOUR COMMANDS HERE`

Reviewer: ```suggestion
- Run locally as part of a Temporal Cluster using [Docker Compose](https://github.com/temporalio/docker-compose): `docker exec temporal-admin-tools tctl YOUR COMMANDS HERE`
```
